docs/guide/adapters/zstack.md

```diff
@@ -180,6 +180,26 @@ Local fast delivery: [Australia](https://shop.dialedin.com.au/products/smlight-s
     |<img src="https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07p7-main1-title.jpg" width="200">|<img src="https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07p7-main2-title.jpg" width="200">|<img src="https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07p7-main3-title.jpg" width="200">|     
 
 </details>
+<details>
+<summary>ZigStar ZBStick-P2 - Zigbee USB Adapter</summary>  
+
+ZigStar is back with upgraded Zigbee Dongle. Developed for Zigbee2MQTT, pre-flashed and ready to use Zigbee adapter.
```

Comment:
Please remove the marketing like text here, I suggest adding the used chip (CC1352P7?)